ASCENT_GUIDANCE.s

```diff
@@ -357,7 +357,7 @@ ASCTERM		EXIT
 		CALL
 			FINDCDUW -2
 ASCTERM1	EXIT
- +1		CA	FLAGWRD9	# INSURE THAT THE NOUN 63 DISPLAY IS
+ +1		CA	FLAGWRD9	# ENSURE THAT THE NOUN 63 DISPLAY IS
```

Comment:
The **Insure** typo is present in the [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0850.jpg)